### PR TITLE
Add variable for PgpDecryption

### PR DESCRIPTION
**Pull request by ejcx**
This PgpPrivate will be used elsewhere

### CODE REVIEW

**Comment by jacobhaven** [comment]
LGTM

**Comment by lziest** [comment]
conflict? :( commit message can be simplified.

File: cli/config.go [hunk 1]
```diff
@@ -96,6 +97,7 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.Path, "path", "/", "Path on which the server will listen")
 	f.StringVar(&c.Password, "password", "0", "Password for accessing PKCS #12 data passed to bundler")
 	f.StringVar(&c.Usage, "usage", "dev", "usage of private key")
+	f.StringVar(&c.PgpPrivate, "pgp-private", "", "file to load a PGP Private key decryption.")
```
Comment (kisom) [inline cli/config.go]: PgpPrivate should be PGPPrivate.

**Comment by ejcx** [comment]
Should be good.

**Comment by kisom** [comment]
LGTM.

**Pull request merged**